lang/src/org/partiql/lang/ast/ExprNodeToStatement.kt

```diff
@@ -19,12 +20,12 @@ fun ExprNode.toAstStatement(): PartiqlAst.Statement {
     }
 }
 
-private fun PartiQlMetaContainer.toElectrolyteMetaContainer(): ElectrolyteMetaContainer =
+internal fun org.partiql.lang.ast.MetaContainer.toIonElementMetaContainer(): IonElementMetaContainer =
```

Comment:
Is the `org.partiql.lang.ast.` part necessary?

Reply:
No, I was trying to fix a build error and left this behind by accident.  Removing...